nvflare/app_common/aggregators/intime_accumulate_model_aggregator.py

```diff
@@ -65,7 +68,7 @@ def accept(self, shareable: Shareable, fl_ctx: FLContext) -> bool:
 
         Returns:
             The first boolean indicates if this shareable is accepted.
-            The second bollean indicates if aggregate can be called.
```

Comment:
For consistency with the other aggregator and since there now is no second boolean, this could just be:

True if contribution is accepted, else False.

Reply:
let's fix this later.